src/index.html

```diff
@@ -15,8 +15,77 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="main-header">
+      <img
+        class="header-left"
+        src="./images/logo.png"
+        alt="moyo"
+      />
+      <nav class="header-right">
```

Comment:
```suggestion
      <nav class="nav">
```
more meaningful naming nav or navigation.
and in nav should be unordered list (ul) with list items.
also suggest such class naming.

Example:
```
 <nav class="nav>
   <ul class="nav__list">
      <li class="nav__item">
        <a class="nav__link is-active">Apple</a>
      </li>
        <li class="nav__item">
          <a class="nav__link">Samsung</a>
      </li>
      ...
   </ul>
 </nav>
```

Reply:
fixed